python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -1233,6 +1233,70 @@ def forward(self, x):
 def ElementwiseDivScalarModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 4))
 
+# ==============================================================================
+
+
+class ElementwiseRemainderScalarModule_Int(torch.nn.Module):
```

Comment:
nit: I would call this test `Elementwise..._Int_Float` and the other int one as `Elementwise..._Int`